apps/component-generator/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "component-generator",
+  "private": true,
+  "version": "1.0.0",
+  "description": "Component-generator is used to automate process of component creation.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
What is this license type, and why are we not using MIT here?

Reply:
It was generated by default. I fixed it